subsys/bluetooth/mesh/main.c

```diff
@@ -223,7 +225,8 @@ void bt_mesh_reset(void)
 		bt_mesh_net_clear();
 	}
 
-	(void)memset(bt_mesh.dev_key, 0, sizeof(bt_mesh.dev_key));
+	bt_mesh_destroy_key(bt_mesh.dev_key);
+	memset(&bt_mesh.dev_key, 0, sizeof(bt_mesh.dev_key));
```

Comment:
```suggestion
	memset(KEY_REF(bt_mesh.dev_key), 0, sizeof(bt_mesh.dev_key));
```
Shouldn't it be `bt_mesh.dev_key = INVALID_KEY_VAL`?

Reply:
removed INVALID_KEY_VAL